idp/docker/Dockerfile.linux.arm64

```diff
@@ -7,7 +7,7 @@ RUN apk update && \
 	echo 'hosts: files dns' >| /etc/nsswitch.conf
 
 LABEL maintainer="ownCloud GmbH <devops@owncloud.com>" \
-  org.label-schema.name="oCIS Konnectd" \
```

Comment:
@butonic Any idea if these labels are referenced anywhere? Is it safe to just rename them?

Reply:
yeah can just be renamed :-)